### DISCUSSION TITLE
Deploing React site with ChackraUI and modal window stopped showing

### DISCUSSION THREAD

**Discussion by BaltsevichDanil** [chakra-ui/chakra-ui#3996] [Q&A] [answered]
I have a page with a modal window.  In development mode, it opens.  But when I upload my site to vercel, then the screen just darkens there, and the modal window does not appear, but all the window controls work. Who faced the same problem?

```
const { isOpen, onOpen, onClose } = useDisclosure()
  return (
    <>
      <Button onClick={onOpen}>Open Modal</Button>

      <Modal isOpen={isOpen} onClose={onClose}>
        <ModalOverlay />
        <ModalContent>
          <ModalHeader>Modal Title</ModalHeader>
          <ModalCloseButton />
          <ModalBody>
            <Lorem count={2} />
          </ModalBody>

          <ModalFooter>
            <Button colorScheme="blue" mr={3} onClick={onClose}>
              Close
            </Button>
            <Button variant="ghost">Secondary Action</Button>
          </ModalFooter>
        </ModalContent>
      </Modal>
    </>
  )
```

**Comment by BaltsevichDanil** [comment]
localhost
![image](https://user-images.githubusercontent.com/35452155/117860069-0dcafc80-b2ba-11eb-8f0f-e5e2dc41e039.png)

vercel
![image](https://user-images.githubusercontent.com/35452155/117860149-29ce9e00-b2ba-11eb-889c-f97794c1fd6f.png)

**Comment by chr-ge** [comment, marked as answer]
I was able to fix this by upgrading the `framer-motion` version -> `"framer-motion": "^4.1.16"`.

**Reply by BaltsevichDanil** [reply]
It helped, thanks!!
